theme/static/js/generic-resource.js

```diff
@@ -535,7 +535,10 @@ $(document).ready(function () {
     // ========================================
     $(window).bind('scroll', function () {
         let toolbar = $(".custom-btn-toolbar");
-        if ($(window).scrollTop() > toolbar_offset && !toolbar.hasClass('toolbar-fixed')) {
+        if (toolbar.children().length == 0){
+            toolbar.css("display", None);
+        }
```

Comment:
Wondering whether you need to remove None display css property when there are toolbar items?

Reply:
This is because the div still showed as a tiny box when there were no icons... I just check for children and turn it off when there are no icons